src/app/components/card/card.component.html

```diff
@@ -0,0 +1,16 @@
+<mat-card class="mat-elevation-z4">
+    <mat-card-title>{{ item.name }}</mat-card-title>
+    <mat-card-content>
+        <button mat-fab color="accent">
+            <mat-icon>arrow_back</mat-icon>
+        </button>
+        <img mat-card-image [src]="item.images[0].url" id="cardImage" />
+        <button mat-fab color="accent">
+            <mat-icon>arrow_forward</mat-icon>
+        </button>
+    </mat-card-content>
+    <mat-card-actions align="end">
+        <button mat-button color="primary"
+        (click)="buyBtnClick.emit($event); $event.stopPropagation()">Купить</button>
```

Comment:
```suggestion
        <button
            mat-button
            color="primary"
            (click)="buyBtnClick.emit($event); $event.stopPropagation()"
        >
            Купить
        </button>
```
Я бы предложил так разнести, читается попроще

\+ Давай остановку всплытия уберем в метод компонента, чтобы шаблон не нагружать

Reply:
Понял, спасибо